frontends/election-manager/src/app_write_in_flows.test.tsx

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import { screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {
+  electionMinimalExhaustiveSampleDefinition,
+  electionMinimalExhaustiveSampleFixtures,
+} from '@votingworks/fixtures';
+import { MemoryCard, MemoryHardware, typedAs } from '@votingworks/utils';
+import { fakeKiosk, hasTextAcrossElements } from '@votingworks/test-utils';
+import fetchMock from 'fetch-mock';
+import { renderRootElement } from '../test/render_in_app_context';
+import { authenticateWithElectionManagerCard } from '../test/util/authenticate';
+import { App } from './app';
+import { ElectionManagerStoreMemoryBackend } from './lib/backends';
+import { VxFiles } from './lib/converters';
+import { MachineConfig } from './config/types';
+
+let mockKiosk!: jest.Mocked<KioskBrowser.Kiosk>;
+
+beforeEach(() => {
+  mockKiosk = fakeKiosk();
+  window.kiosk = mockKiosk;
+  fetchMock.reset();
+  fetchMock.get(
+    '/convert/tallies/files',
+    typedAs<VxFiles>({
+      inputFiles: [{ name: 'name' }, { name: 'name' }],
+      outputFiles: [{ name: 'name' }],
+    })
+  );
+  fetchMock.get(
+    '/machine-config',
+    typedAs<MachineConfig>({
+      machineId: '0000',
+      codeVersion: 'TEST',
+    })
+  );
+  fetchMock.delete('/admin/write-ins/cvrs', { body: { status: 'ok ' } });
+});
+
+afterEach(() => {
+  delete window.kiosk;
+});
+
+test('manual write-in data end-to-end test', async () => {
```

Comment:
I chose to break out this test from the `app.test.tsx` file because it is getting unwieldy, but we can move it back in it preferred.

Reply:
No seperate is better IMO